internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/NettyP2PBlobTransferManager.java

```diff
@@ -0,0 +1,72 @@
+package com.linkedin.venice.blobtransfer;
```

Comment:
I made a few modifications that ishwarya asked for in the blob transfer file when it was in the blob package, when i push will it make a copy in the blob package that will then need to be transferred to the blobtransfer package or will we have to manually move it to the blob transfer package after my commit is merged?

Reply:
This PR is not merged yet so there's no conflict yet. However, if you want to move your files from `blob` to `blobtransfer` manually in your PR, it'd be good too. I can pull your change and resolve conflicts if any.

Reply:
sounds good, I'll do that now just to be safe